stdlib/task/src/main/ballerina/src/task/scheduler.bal

```diff
@@ -98,8 +98,8 @@ public type Scheduler object {
 
     # Checks whether the task listener is started or not.
     #
-    # + return - Returns `true` if the Scheduler is already started, `false` if the Scheduler is
-    #               not started yet or stopped calling the `Scheduler.stop()` function.
+    # + return - `true` if the `Scheduler` is already started, `false` if the `Scheduler` is
```

Comment:
```suggestion
    # + return - `true` if the `Scheduler` is already started, `false` if the `Scheduler` is
```
Same as above. Also, please refer the template for the correct format of the return statement description.